### PR TITLE
[ANCHOR-704] Clean up `amount_fee` and document `user_action_required_by`

### PR DESCRIPTION
**Pull request by JiahuiWho**
This is a resubmit of https://github.com/stellar/stellar-docs/pull/857 due to rebase

1. Clean up deprecated amount_fee and update platform transaction schema
2. Add user_action_required_by to related RPC methods
NotifyInteractiveFlowCompletedRequest
NotifyTransactionOnHoldRequest
NotifyOffchainFundsPendingRequest
NotifyOffchainFundsAvailableRequest
RequestCustomerInfoUpdateRequest
RequestOffchainFundsRequest
RequestOnchainFundsRequest
RequestTrustRequest
3. Add use instruction in SEP-6 and SEP-24
platforms/anchor-platform/admin-guide/sep6/configuration.mdx
platforms/anchor-platform/admin-guide/sep24/configuration.mdx

### CODE REVIEW

**Comment by ElliotFriend** [comment]
Looks good from what I can tell. Just want to make sure, though, this is still needed for `v2.x` anchor platform, right?

File: openapi/anchor-platform/bundled-callbacks.yaml
Comment (JakeUrban) [inline openapi/anchor-platform/bundled-callbacks.yaml:1094]: Did we remove `amount_fee` from the response we return? I think it should still be there to avoid breaking wallets that still use it. It should just always equal `fee.total`

File: platforms/anchor-platform/admin-guide/sep24/configuration.mdx
Comment (JakeUrban) [inline platforms/anchor-platform/admin-guide/sep24/configuration.mdx:117]: I must have missed the PR implementing `user_action_required_by` so I'm missing context, but why do we have a configurable default value for this? Wouldn't the time the user has to act be highly dependent on the specific situation? I would've thought we would always require the business to set the time in each RPC request.

Also when you say "automatically moves to the next status", the business still has to make an RPC request to move the transaction to the next status right? The anchor platform doesn't monitor the time and move the transaction status on its own.

**Comment by Ifropc** [comment]
> Just want to make sure, though, this is still needed for v2.x anchor platform, right? 

Yes, I asked Jiahui to wait a bit with this PR to not mess up your reorganizing PR 🙏🏼

File: platforms/anchor-platform/admin-guide/sep24/configuration.mdx
Comment (Ifropc) [inline platforms/anchor-platform/admin-guide/sep24/configuration.mdx:103]: Is more_info_base_url not mandatory for sep-24 anymore?

File: openapi/anchor-platform/bundled-callbacks.yaml
Comment (JiahuiWho) [reply openapi/anchor-platform/bundled-callbacks.yaml:1094]: Right `amount_fee` is removed in 3.0 https://stellarorg.atlassian.net/browse/ANCHOR-636 as breaking change

File: platforms/anchor-platform/admin-guide/sep6/configuration.mdx
Comment (Ifropc) [inline platforms/anchor-platform/admin-guide/sep6/configuration.mdx:183]: This methods are not alternative (mutually exclusive), they should be used in combination. Can you reword that?
`Alternatively` -> `In addition,`
And let's not use "they" (thinking of a person or people), we can use business/anchor or passive voice (thinking of a server).

File: platforms/anchor-platform/admin-guide/sep24/configuration.mdx
Comment (Ifropc) [reply platforms/anchor-platform/admin-guide/sep24/configuration.mdx]: Do you min adding an example of GET /transactions usage?

**Review by Ifropc — CHANGES_REQUESTED** [review]
Let's wait until [versioning pr is merged](https://github.com/stellar/stellar-docs/pull/909)

**Comment by Ifropc** [comment]
@JiahuiWho we probably want to start this PR fresh off the current main branch

**Pull request closed without merging**
